megamek/src/megamek/client/ui/swing/BoardEditor.java

```diff
@@ -903,6 +905,27 @@ private void setupEditorPanel() {
         choTerrainType.addActionListener(e -> {
             if (!terrListBlocker) {
                 lisTerrain.clearSelection();
+                
+                // if we've selected DEPLOYMENT ZONE, disable the "exits" buttons
+                // and make the "exits" popup point to a multi-select list that lets
+                // the user choose which deployment zones will be flagged here
+                
+                // otherwise, re-enable all the buttons and reset the "exits" popup to its normal behavior
+                if (((TerrainHelper) choTerrainType.getSelectedItem()).getTerrainType() == Terrains.DEPLOYMENT_ZONE) {
+                    butExitUp.setEnabled(false);
+                    butExitDown.setEnabled(false);
+                    texTerrExits.setEnabled(false);
+                    cheTerrExitSpecified.setEnabled(false);
+                    cheTerrExitSpecified.setText("Zones");//Messages.getString("BoardEditor.deploymentZoneIDs"));
```

Comment:
For some reason, changing the label text for the "exits" checkbox causes the board editor panel to expand, not really sure how to address it.

Reply:
Just tested it, didn't expand for me.